src/relay/op/tensor/transform.cc

```diff
@@ -673,6 +675,17 @@ bool FullRel(const Array<Type>& types,
   return true;
 }
 
+Array<Tensor> FullCompute(const Attrs& attrs,
+                          const Array<Tensor>& inputs,
```

Comment:
@jroesch i hav some confusion here. seems the behavior of full/full_like is different from numpy. this wil always fill 0 only.
[This pr ](https://github.com/dmlc/tvm/pull/2170)is according to numpy impl.

Reply:
This is probably just a bug from when I was hacking, I'm happy to replace my implementation with yours. I was just going top-down filling in ops as needed while working on frontend.